frame/democracy/src/lib.rs

```diff
@@ -364,11 +399,10 @@ decl_module! {
 		/// # </weight>
 		#[weight = SimpleDispatchInfo::FixedNormal(5_000_000)]
```

Comment:
Maybe update `weight`, a transfer is `1_000_000`.

Reply:
i'd prefer to address all weights in a separate PR.